<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainmenulogin</class>
 <widget class="QWidget" name="mainmenulogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#createRoom{
	background: #494F55;
}

#joinRoom{
	background: #494F55;
}


#label{
	color: white;
}

#label_3{
	color: white;
}

#create{
	
}

#roomNameInput{
	background: none;
	border: none;
}</string>
  </property>
  <widget class="QFrame" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>510</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	font-size: 12px;
	font-family: century;
}

#background{
	background-image: url(:/background.jpg);
}

#header{
	font-size: 40px;
}

QPushButton{
	background: transparent;
	border:none;
	color: white;
	font-size: 20px;
	text-align: left;
}

#OnlineList{
	
}

#logout{
	text-align: right;
}

#OnlineList{
	background: white;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">transform: rotate(90);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource/user_icon.qrc">
      <normaloff>:/right-from-bracket-solid.svg</normaloff>:/right-from-bracket-solid.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateRoom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Create Room</string>
    </property>
   </widget>
   <widget class="QPushButton" name="JoinRoom">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Join Room</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LeaderBoard">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>470</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Leader Board</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditProfile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Edit Profile</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="createRoom">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>400</width>
     <height>150</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert room name to create</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roomNameInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>350</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLabel" name="warning_create">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;
font-size: 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="joinRoom">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>400</width>
     <height>150</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert room name to join</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="joinInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>350</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="join">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Join</string>
    </property>
   </widget>
   <widget class="QLabel" name="warning_join">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px;
color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>400</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #494F55;
color: white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>USERNAME: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>INGAME: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>WINS: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LOSES: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ELO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>WINRATE: </string>
    </property>
   </widget>
   <widget class="QLabel" name="username">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ingame">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="wins">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="loses">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="winrate">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
border: none;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
  <widget class="QListView" name="list_player">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>300</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <zorder>list_player</zorder>
  <zorder>background</zorder>
  <zorder>createRoom</zorder>
  <zorder>joinRoom</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources>
  <include location="Resource/user_icon.qrc"/>
 </resources>
 <connections/>
</ui>
